app/Services/Database/TrackService.php

```
@@ -0,0 +1,47 @@
+    /**
+     * Sync artists to a track.
+     */
+    public function syncArtists(Track $track, array $artistIds): void
+    {
+        $track->artists()->sync($artistIds);
+    }
+
+    /**
+     * Sync track to a playlist with position.
+     */
+    public function syncToPlaylist(Track $track, int $playlistId, int $position): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$playlistId'. (undefined)

(UnusedFormalParameter)